packages/next/src/build/swc/index.ts

```diff
@@ -879,7 +879,7 @@ function bindingToApi(
     nextConfigSerializable.exportPathMap = {}
     nextConfigSerializable.webpack = nextConfig.webpack && {}
 
-    if (nextConfigSerializable.experimental?.turbo?.rules) {
+    if (nextConfigSerializable.turbopack?.rules) {
```

Comment:
I checked the git blame, and this was clearly just a typo when migrating from the experimental config.